src/pages/Donate/DonateSupportUsRightRail.vue

```diff
@@ -69,7 +70,7 @@ export default {
 				item => item.key === 'web-donate-support-us-callout-2'
 			) || {};
 			const fundInnovationImage = this.content?.media?.find(
-				image => image.title === 'Globe Grid Illustration'
+				image => image.title === 'homepage-stats-icon-3'
 			);
```

Comment:
This illustration from contentful was a different size than the piggy bank. It had transparent padding within the image itself.
![image](https://user-images.githubusercontent.com/187937/141218082-39b14597-2b4d-42ed-acbf-019be0ff7ae9.png)

The new one is a touch different but lines up correctly 
![image](https://user-images.githubusercontent.com/187937/141218128-11ba4469-7360-439a-8d7c-e9453bcedf18.png)

Reply:
Thank you! I think that image was potentially changed for a different context after it was initially used here...